### PR TITLE
Create and style Footer component

### PR DESCRIPTION
**Pull request by l-goede**
Create and style footer
http://localhost:3331/storybook/?path=/story/component-footer--nav-bar

### CODE REVIEW

File: src/app/Components/SampleButton/SampleButton.module.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.button {
+  color: black;
+  background: white;
+  font-size: 0.8em;
+  padding: 1em 1.5em;
+  text-transform: uppercase;
+  width: 30%;
+  font-weight: bold;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```
Comment (J-Bannach) [inline src/app/Components/SampleButton/SampleButton.module.css:9]: Is this the right font family?

File: src/app/Components/SampleButton/SampleButton.module.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  height: 3.2em;
+  border: black 2px solid;
```
Comment (J-Bannach) [inline src/app/Components/SampleButton/SampleButton.module.css:12]: Maybe use em instead?

File: src/app/Components/SearchField/SearchField.module.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  font-family: var(--primary-font);
+  color: var(--secondary-text-color);
+  letter-spacing: 0.08em;
+}
+
+.container__input {
+  border: none;
+  background: var(--input-color);
+  border-radius: 2.5em;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.16);
```
Comment (J-Bannach) [inline src/app/Components/SearchField/SearchField.module.css:17]: Maybe use em instead of px?

**Review by J-Bannach — APPROVED** [review]
lgtm 🍰

**Review by Maurizio-F — APPROVED** [review]
lgtm! 🚀

**Pull request merged**